docs/how-to-add-a-new-script-to-the-cea.rst

```diff
@@ -1,16 +1,15 @@
-Contributing
-=============
+How to add a new script
```

Comment:
ah. i think this explains my previous question :-)

Reply:
:-)